src/ResourceManagement/Logic/Logic.Tests/ScenarioTests/IntegrationAccountSessions.ScenarioTests.cs

```diff
@@ -66,15 +66,12 @@ public void CreateAndDeleteIntegrationAccountSession()
                     integrationAccountName: integrationAccountName,
                     sessionName: integrationAccountSessionName);
 
-                // Getting an absent record should not throw, RP returns 404 and error response
-                var errorResponse = client.Sessions.Get(
-                    resourceGroupName: Constants.DefaultResourceGroup,
-                    integrationAccountName: integrationAccountName,
-                    sessionName: integrationAccountSessionName) as ErrorResponse;
-                Assert.NotNull(errorResponse);
-                Assert.NotNull(errorResponse.Error);
-                Assert.Equal(expected: "SessionNotFound", actual: errorResponse.Error.Code);
-                Assert.Contains(expectedSubstring: "could not be found in integration account", actualString: errorResponse.Error.Message);
```

Comment:
The asserts are removed because the SDK behavior is changed. The existing asserts were validating that an error response object is returned. However that was not the proper behavior. With the swagger fix the new and proper behavior is than an error response exception is thrown. The new assert validates that new and proper behavior.

Note per the PR title and notes that this is a breaking change hence regression tests have to be updated.